llm/app/core/redis_manager.py

```
@@ -109,35 +176,49 @@ def publish_data_request(self, request_data: Dict[str, Any]) -> str:
+                    messages = await self.redis_client.xreadgroup(
+                        self.settings.redis_consumer_group,
+                        self.consumer_name,  # 고유한 consumer name 사용
+                        {self.settings.redis_stream_data_results: ">"},
+                        count=10,
+                        block=1000  # 1초 대기
+                    )
 
-                for stream, stream_messages in messages:
-                    for message_id, fields in stream_messages:
+                    # 읽어온 메시지들을 unacked_messages에 추가
+                    for stream, stream_messages in messages:
```

Ruff (0.12.2)

Loop control variable `stream` not used within loop body

Rename unused `stream` to `_stream`

(B007)